app/src/main/java/com.example.weatherapp/utils/Constants.kt

```diff
@@ -3,4 +3,5 @@ package com.example.weatherapp.utils
 object Constants {
     const val API_PATTERN = "yyyy-MM-dd HH:mm:ss"
     const val APP_PATTERN = "EEE, MM/dd HH:mm"
+    const val TEMPERATURE_UNIT = "°C"
```

Comment:
as you use this constant in the adapter to show it on the screen, this should be in the string resources

Reply:
Moved temperature_unit from Constants to strings resource.